.one-pipeline.yaml

```diff
@@ -227,46 +223,39 @@ containerize:
         set -x
       fi
 
-      # Download and configure golang
-      GO_VERSION=$(get_env go-version)
-      if [[ -z "${GO_VERSION}" ]]; then
-        GO_VERSION="$(grep '^go [0-9]\+.[0-9]\+' go.mod | cut -d ' ' -f 2)"
-      fi
-      rm -rf /usr/local/go && wget --no-verbose --header "Accept: application/octet-stream" "https://golang.org/dl/go${GO_VERSION}.linux-amd64.tar.gz" -O - | tar -xz -C /usr/local/
+      ## Setup required tooling
+      make setup-go GO_RELEASE_VERSION=$(get_env go-version)
       export PATH=$PATH:/usr/local/go/bin
+      yum -y -q update
 
-      apt-get update
-      apt-get -qq -y install build-essential software-properties-common uidmap
       #  Build images
       export RELEASE_TARGET=$(get_env branch)
       
       if [[ -z $DISABLE_ARTIFACTORY ]]; then
-        DISABLE_ARTIFACTORY="false"
+        export DISABLE_ARTIFACTORY="false"
       fi
       if [[ "$DISABLE_ARTIFACTORY" != "false" ]]; then
```

Comment:
@halim-lee This is unrelated to your change. But it'll be good to get rid of this 'big' if-else check that duplicates the steps for non-artifactory. Instead, have smaller if-checks around artifactory specific steps. I am fine with addressing it via a separate PR.

Reply:
I will open a separate PR for that change!

Reply:
Sounds good. 

I also merged https://github.ibm.com/websphere/operators/pull/13, so the branch can be switched to main. Thank you.